thinc/config.py

```diff
@@ -1,5 +1,6 @@
 from typing import Union, Dict, Any, Optional, List, Tuple, Callable, Type, Sequence
 from types import GeneratorType
+from .types import Decorator
```

Comment:
This should be moved down onto line 14 to separate absolute from  relative imports